### DISCUSSION TITLE
Community Edition accepts undefined streams after restart

### DISCUSSION THREAD

**Discussion by jpstruhar** [ant-media/Ant-Media-Server#6847] [Q&A]
I'm running a Community Edition server via DigitalOcean. I have 3 cameras that are configured to connect via RTMP any time a stream is available. I use a Cron job to make an API call to start and stop streams on a schedule. When a stream is started via the API call, the camera connects, the stream starts, all is right.
However, if I shut down the server and reboot the virtual machine, when it comes back up it starts accepting undefined streams and the cameras connect and start streaming. I can't figure out why this is happening. If I check the 'undefined streams' box, save, uncheck the box, save again, it will again behave as expected.
Any insight as to why this isn't saved through a reboot? TIA for any help with this.

**Comment by amarantmeida** [comment]
@jpstruhar If Dynamic Stream Pulling is enabled, Ant Media Server could automatically detect the available streams and start pulling them even after a reboot, leading to the undefined streams behaviour you’re seeing. This could override manual configurations or API calls you’ve set up to control which streams should be active.
To enable/disable the feature for the stream, check the **Auto Start/Stop Streaming** checkbox as shown below.
![image](https://github.com/user-attachments/assets/946dbc1c-0acf-4014-becb-9d9e6c83ec19)

**Comment by jpstruhar** [comment]
Thank you. I see that this is available if I am creating a new 'Stream Source' but I am not. I am creating a 'Live Stream' which creates an rtmp endpoint for my cameras to connect to rather than pulling an rtsp. It is not an available option when creating a 'Live Stream.'
